src/GitHub.Exports/Services/IGitService.cs

```diff
@@ -44,5 +45,7 @@ public interface IGitService
         /// <param name="repo"></param>
         /// <returns></returns>
         UriString GetRemoteUri(IRepository repo, string remote = "origin");
+
+        Task<string> GetLatestPushedSha(string path);
```

Comment:
You'll need to copy the documentation added to the method to here. Classes and interfaces don't share docs, unfortunately.